lib/install/importmap.rb

```diff
@@ -0,0 +1,74 @@
+templates_path = File.expand_path("../generators/stimulus_reflex/templates", File.join(File.dirname(__FILE__)))
+
+importmap_src = templates_path + "/config/importmap.rb.tt"
+importmap_path = Rails.root.join("config/importmap.rb")
+
+friendly_importmap_path = importmap_path.relative_path_from(Rails.root).to_s
+if importmap_path.exist?
+  if File.read(importmap_path) == File.read(importmap_src)
+    say "✅ #{friendly_importmap_path} is present"
+  else
+    copy_file(importmap_path, "#{importmap_path}.bak", verbose: false)
+    remove_file(importmap_path, verbose: false)
+    copy_file(importmap_src, importmap_path, verbose: false)
+    append_file("tmp/stimulus_reflex_installer/backups", "#{friendly_importmap_path}\n", verbose: false)
+    say "#{friendly_importmap_path} has been created"
+    say "❕ original importmap.rb renamed importmap.rb.bak", :green
+  end
+else
+  copy_file(importmap_src, importmap_path)
+end
```

Comment:
We don't need to copy the `importmap` file into the app. Rails is smart enough to merge the pinned modules by itself, because of this:
https://github.com/stimulusreflex/stimulus_reflex/blob/master/lib/stimulus_reflex/engine.rb#L22-L26

Reply:
Did this because I could not figure out how to access `pin` and `pin_all_from`, as I said previously in this thread and on Discord. Any help is appreciated! I would vastly prefer to just call these methods instead of copying.

Reply:
If you remove this could it should still just work

Reply:
Just to re-state: I would be thrilled to be able to add both `pin` and `pin_all_from` lines using Thor actions. I still have not been able to figure out how to call the importmap Thor actions, so I'm going to convert all of the `pin` requirements to `system "bin/importmap pin X"` calls and I guess I'll use `append_file` for the `pin_all_from` directives as there doesn't seem to be a CLI command for these.

If you know how to call `pin` from within these templates, I'd love to switch away from `system` calls.